doc/source/io.rst

```diff
@@ -3941,56 +3961,165 @@ destination DataFrame as well as a preferred column order as follows:
+Writing large DataFrames can result in errors due to size limitations being exceeded.
+This can be avoided by setting the ``chunksize`` argument when calling :func:`~pandas.io.to_gbq`.
+For example, the following writes ``df`` to a BigQuery table in batches of 10000 rows at a time:
 
-You can access the management console to determine project id's by:
-<https://code.google.com/apis/console/b/0/?noredirect>
+.. code-block:: python
+
+    pd.to_gbq(df, 'my_dataset.my_table', projectid, chunksize=10000)
 
-As of 0.15.2, the gbq module has a function ``generate_bq_schema`` which
-will produce the dictionary representation of the schema.
+You can also see the progess of your post via the ``verbose`` flag which defaults to ``True``.
+For example:
 
 .. code-block:: python
 
-   df = pandas.DataFrame({'A': [1.0]})
-   gbq.generate_bq_schema(df, default_type='STRING')
+    pd.to_gbq(df, 'my_dataset.my_table', projectid, chunksize=10000, verbose=True)
 
-.. warning::
+.. note::
 
-   To use this module, you will need a valid BigQuery account. See
-   <https://cloud.google.com/products/big-query> for details on the
-   service.
+    The http response code of Google BigQuery can be successful (200) even if the append failed.
+    For this reason, if there is a failure to append to the table, the complete error response
+    from BigQuery is returned which can be quite long given it provides a status for each row.
+    You may want to start with smaller chunks to test that the size and types of your dataframe
+    match your destination table to make debugging simpler.
+
+.. note::
+
+    The BigQuery SQL query language has some oddities, see
+    `here <https://developers.google.com/bigquery/query-reference>`__
+
+.. note::
+
+    While BigQuery uses SQL-like syntax, it has some important differences from traditional
+    databases both in functionality, API limitations (size and quantity of queries or uploads),
+    and how Google charges for use of the service. You should refer to Google documentation
+    often as the service seems to be changing and evolving. BiqQuery is best for analyzing large
+    sets of data quickly, but it is not a direct replacement for a transactional database.
+
+
+Creating BigQuery Tables
+''''''''''''''''''''''''
+
+As of 0.17.0, the gbq module has a function :func:`~pandas.io.create_table` which allows users
+to create a table in bigquery. The only requirement is that the dataset must already exsit.
+The schema may be generated from a dataframe using the :func:`~pandas.io.generate_bq_schema` function below.
+
+For example:
+
+.. code-block:: python
+
+    pd.create_table('my_dataset.my_table', schema, projectid)
+
+As of 0.15.2, the gbq module has a function :func:`~pandas.io.generate_bq_schema` which will
+produce the dictionary representation schema of the specified pandas DataFrame.
+
+.. code-block:: python
+
+   df = pd.DataFrame({'A': [1.0]})
+   pd.generate_bq_schema(df, default_type='STRING')
```

Comment:
use your example df from above (which has multiple dtypes) and show the output here (it's a code block so paste it in)